content/news/2021-04-21-some-updates/index.id.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Pembaruan Ubucon Asia 2021"
+date: 2021-04-21T00:10:16+09:00
+draft: false
+authors:
+    - name: Youngbin Han
+      bio: Penyelenggara, Ubucon Asia 2021/Penasehat, Komunitas Ubuntu Korea 
+      email: youngbin@ubuntu-kr.org
+      launchpad: sukso96100
+      github: sukso96100
+      profile: https://avatars2.githubusercontent.com/u/1916739?s=460&v=4
+      linkurl: https://youngbin.xyz
+      linklabel: Website
+---
+
+Sudah sekitar 3 bulan setelah saya menerbitkan posting ["Mencari penyelenggara"](../ 2021-01-07-mencari-penyelenggara).
```

Comment:
```suggestion
Sudah sekitar 3 bulan setelah saya menerbitkan posting ["Mencari penyelenggara"](../2021-01-07-looking-for-organizers).
```
If you have not localized slug for the post `2021-01-07-looking-for-organizers` relative path to the post should be same as from english one.